pkg/jaeger/query/trace_query.go

```diff
@@ -220,12 +241,14 @@ func buildTraceIDSubquery(q *spanstore.TraceQueryParameters) (string, []interfac
 
 	}
 
-	query := ""
+	clauseString := ""
 	if len(clauses) > 0 {
-		query = fmt.Sprintf(subqueryFormat, strings.Join(clauses, " AND "))
+		clauseString = strings.Join(clauses, " AND ")
 	} else {
-		query = fmt.Sprintf(subqueryFormat, "TRUE")
+		clauseString = "TRUE"
 	}
+	params = append(params, b.cfg.MaxTraceDuration)
+	query := fmt.Sprintf(subqueryFormat, len(params), clauseString)
```

Comment:
so I think that is correct since its a single parameter with an interval. If it's parameter 4 this would become 
```
start_time_max - $4::interval as time_low, start_time_max + $4::interval as time_high
```
another words we are adding and subtracting the same interval

Reply:
Oh, I see. Could you add a comment? Splicing a numbered prepared parameter with a numbered Sprintf parameter is too terse for me 😅